be/src/fs/fs_starlet.cpp

```diff
@@ -33,22 +33,12 @@ static Status to_status(absl::Status absl_status) {
 }
 
 static const char* const kStarletPrefix = "staros://";
```

Comment:
two const?

Reply:
one is meaning kStarletPrefix is type of const char * . one is for kStarletPrefix itself